editor/quick_open.h

```diff
@@ -28,8 +28,8 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#ifndef EDITOR_QUICK_OPEN_H
-#define EDITOR_QUICK_OPEN_H
+#ifndef QUICK_OPEN_H
+#define QUICK_OPEN_H
```

Comment:
Perhaps the file should be renamed to `editor_quick_open.h`?

Reply:
Yes, done.